### ISSUE TITLE
2025-03-27, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#122]
# Today's Arxiv Papers

(1) [Mobile-MMLU: A Mobile Intelligence Language Understanding Benchmark](http://arxiv.org/abs/2503.20786v1), 2025-03-26

(2) [Understanding R1-Zero-Like Training: A Critical Perspective](http://arxiv.org/abs/2503.20783v1), 2025-03-26

(3) [Feature4X: Bridging Any Monocular Video to 4D Agentic AI with Versatile Gaussian Feature Fields](http://arxiv.org/abs/2503.20776v1), 2025-03-26

(4) [MCTS-RAG: Enhancing Retrieval-Augmented Generation with Monte Carlo Tree Search](http://arxiv.org/abs/2503.20757v1), 2025-03-26

(5) [Beyond Believability: Accurate Human Behavior Simulation with Fine-Tuned LLMs](http://arxiv.org/abs/2503.20749v1), 2025-03-26

(6) [MATHGLANCE: Multimodal Large Language Models Do Not Know Where to Look in Mathematical Diagrams](http://arxiv.org/abs/2503.20745v1), 2025-03-26

(7) [Dynamic Motion Blending for Versatile Motion Editing](http://arxiv.org/abs/2503.20724v1), 2025-03-26

(8) [From Annotation to Adaptation: Metrics, Synthetic Data, and Aspect Extraction for Aspect-Based Sentiment Analysis with Large Language Models](http://arxiv.org/abs/2503.20715v1), 2025-03-26

(9) [GLRD: Global-Local Collaborative Reason and Debate with PSL for 3D Open-Vocabulary Detection](http://arxiv.org/abs/2503.20682v1), 2025-03-26

(10) [Vision as LoRA](http://arxiv.org/abs/2503.20680v1), 2025-03-26

(11) [Mitigating Low-Level Visual Hallucinations Requires Self-Awareness: Database, Model and Training Strategy](http://arxiv.org/abs/2503.20673v1), 2025-03-26

(12) [TAMA: A Human-AI Collaborative Thematic Analysis Framework Using Multi-Agent LLMs for Clinical Interviews](http://arxiv.org/abs/2503.20666v1), 2025-03-26

(13) [TN-Eval: Rubric and Evaluation Protocols for Measuring the Quality of Behavioral Therapy Notes](http://arxiv.org/abs/2503.20648v1), 2025-03-26

(14) [Unlocking Efficient Long-to-Short LLM Reasoning with Model Merging](http://arxiv.org/abs/2503.20641v1), 2025-03-26

(15) [Collaborative Storytelling and LLM: A Linguistic Analysis of Automatically-Generated Role-Playing Game Sessions](http://arxiv.org/abs/2503.20623v1), 2025-03-26

(16) [What to Retrieve for Effective Retrieval-Augmented Code Generation? An Empirical Study and Beyond](http://arxiv.org/abs/2503.20589v1), 2025-03-26

(17) [Synthetic Data Augmentation for Cross-domain Implicit Discourse Relation Recognition](http://arxiv.org/abs/2503.20588v1), 2025-03-26

(18) [Is Reuse All You Need? A Systematic Comparison of Regular Expression Composition Strategies](http://arxiv.org/abs/2503.20579v1), 2025-03-26

(19) [LLPut: Investigating Large Language Models for Bug Report-Based Input Generation](http://arxiv.org/abs/2503.20578v1), 2025-03-26

(20) [Optimizing Case-Based Reasoning System for Functional Test Script Generation with Large Language Models](http://arxiv.org/abs/2503.20576v1), 2025-03-26

(21) [Low-resource Information Extraction with the European Clinical Case Corpus](http://arxiv.org/abs/2503.20568v1), 2025-03-26

(22) [A Theoretical Framework for Prompt Engineering: Approximating Smooth Functions with Transformer Prompts](http://arxiv.org/abs/2503.20561v1), 2025-03-26

(23) [Injecting Adrenaline into LLM Serving: Boosting Resource Utilization and Throughput via Attention Disaggregation](http://arxiv.org/abs/2503.20552v1), 2025-03-26

(24) [Knowledge-Based Multi-Agent Framework for Automated Software Architecture Design](http://arxiv.org/abs/2503.20536v1), 2025-03-26

(25) [StableToolBench-MirrorAPI: Modeling Tool Environments as Mirrors of 7,000+ Real-World APIs](http://arxiv.org/abs/2503.20527v1), 2025-03-26

(26) [Exploring the Effect of Robotic Embodiment and Empathetic Tone of LLMs on Empathy Elicitation](http://dx.doi.org/10.1007/978-981-96-3525-2_1), 2025-03-26

(27) [Explainable ICD Coding via Entity Linking](http://arxiv.org/abs/2503.20508v1), 2025-03-26

(28) [Vision-Amplified Semantic Entropy for Hallucination Detection in Medical Visual Question Answering](http://arxiv.org/abs/2503.20504v1), 2025-03-26

(29) [MLLM-Selector: Necessity and Diversity-driven High-Value Data Selection for Enhanced Visual Instruction Tuning](http://arxiv.org/abs/2503.20502v1), 2025-03-26

(30) [VPO: Aligning Text-to-Video Generation Models with Prompt Optimization](http://arxiv.org/abs/2503.20491v1), 2025-03-26

(31) [From Trial to Triumph: Advancing Long Video Understanding via Visual Context Sample Scaling and Self-reward Alignment](http://arxiv.org/abs/2503.20472v1), 2025-03-26

(32) [Attention Xception UNet (AXUNet): A Novel Combination of CNN and Self-Attention for Brain Tumor Segmentation](http://arxiv.org/abs/2503.20446v1), 2025-03-26

(33) [RALLRec+: Retrieval Augmented Large Language Model Recommendation with Reasoning](http://arxiv.org/abs/2503.20430v1), 2025-03-26

(34) [CFunModel: A "Funny" Language Model Capable of Chinese Humor Generation and Processing](http://arxiv.org/abs/2503.20417v1), 2025-03-26

(35) [MoLe-VLA: Dynamic Layer-skipping Vision Language Action Model via Mixture-of-Layers for Efficient Robot Manipulation](http://arxiv.org/abs/2503.20384v1), 2025-03-26

(36) [UB-Mesh: a Hierarchically Localized nD-FullMesh Datacenter Network Architecture](http://arxiv.org/abs/2503.20377v1), 2025-03-26

(37) [Dewey Long Context Embedding Model: A Technical Report](http://arxiv.org/abs/2503.20376v1), 2025-03-26

(38) [Dynamic Pyramid Network for Efficient Multimodal Large Language Model](http://arxiv.org/abs/2503.20322v1), 2025-03-26

(39) [Iterative Prompting with Persuasion Skills in Jailbreaking Large Language Models](http://arxiv.org/abs/2503.20320v1), 2025-03-26

(40) [Instruction-Oriented Preference Alignment for Enhancing Multi-Modal Comprehension Capability of MLLMs](http://arxiv.org/abs/2503.20309v1), 2025-03-26

(41) [A Multilingual, Culture-First Approach to Addressing Misgendering in LLM Applications](http://arxiv.org/abs/2503.20302v1), 2025-03-26

(42) [QualiSpeech: A Speech Quality Assessment Dataset with Natural Language Reasoning and Descriptions](http://arxiv.org/abs/2503.20290v1), 2025-03-26

(43) [sudo rm -rf agentic_security](http://arxiv.org/abs/2503.20279v1), 2025-03-26

(44) [The cell as a token: high-dimensional geometry in language models and cell embeddings](http://arxiv.org/abs/2503.20278v1), 2025-03-26

(45) [ViLBench: A Suite for Vision-Language Process Reward Modeling](http://arxiv.org/abs/2503.20271v1), 2025-03-26

(46) [L4: Diagnosing Large-scale LLM Training Failures via Automated Log Analysis](http://arxiv.org/abs/2503.20263v1), 2025-03-26

(47) [LGR: LLM-Guided Ranking of Frontiers for Object Goal Navigation](http://arxiv.org/abs/2503.20241v1), 2025-03-26

(48) [TeleLoRA: Teleporting Model-Specific Alignment Across LLMs](http://arxiv.org/abs/2503.20228v1), 2025-03-26

(49) [Raising Awareness of Location Information Vulnerabilities in Social Media Photos using LLMs](http://dx.doi.org/10.1145/3706598.3714074), 2025-03-26

(50) [Qwen2.5-Omni Technical Report](http://arxiv.org/abs/2503.20215v1), 2025-03-26

(51) [SARGes: Semantically Aligned Reliable Gesture Generation via Intent Chain](http://arxiv.org/abs/2503.20202v1), 2025-03-26

(52) [Open Deep Search: Democratizing Search with Open-source Reasoning Agents](http://arxiv.org/abs/2503.20201v1), 2025-03-26

(53) [Enhancing the Robustness of LLM-Generated Code: Empirical Study and Framework](http://arxiv.org/abs/2503.20197v1), 2025-03-26

(54) [GAPO: Learning Preferential Prompt through Generative Adversarial Policy Optimization](http://arxiv.org/abs/2503.20194v1), 2025-03-26

(55) [Cross-Modal Prototype Allocation: Unsupervised Slide Representation Learning via Patch-Text Contrast in Computational Pathology](http://arxiv.org/abs/2503.20190v1), 2025-03-26

(56) [Rethinking Vision-Language Model in Face Forensics: Multi-Modal Interpretable Forged Face Detector](http://arxiv.org/abs/2503.20188v1), 2025-03-26

(57) [Leveraging Implicit Sentiments: Enhancing Reliability and Validity in Psychological Trait Evaluation of LLMs](http://arxiv.org/abs/2503.20182v1), 2025-03-26

(58) [Can We Make Code Green? Understanding Trade-Offs in LLMs vs. Human Code Optimizations](http://arxiv.org/abs/2503.20126v1), 2025-03-26

(59) [Synthesizing world models for bilevel planning](http://arxiv.org/abs/2503.20124v1), 2025-03-26

(60) [Efficient Model Development through Fine-tuning Transfer](http://arxiv.org/abs/2503.20110v1), 2025-03-25

(61) [Direct Post-Training Preference Alignment for Multi-Agent Motion Generation Models Using Implicit Feedback from Pre-training Demonstrations](http://arxiv.org/abs/2503.20105v1), 2025-03-25

(62) [Bigger But Not Better: Small Neural Language Models Outperform Large Language Models in Detection of Thought Disorder](http://arxiv.org/abs/2503.20103v1), 2025-03-25

(63) [MatplotAlt: A Python Library for Adding Alt Text to Matplotlib Figures in Computational Notebooks](http://dx.doi.org/10.1111/cgf.70119), 2025-03-25

(64) [Generative Linguistics, Large Language Models, and the Social Nature of Scientific Success](http://arxiv.org/abs/2503.20088v1), 2025-03-25

(65) [Can Multi-modal (reasoning) LLMs work as deepfake detectors?](http://arxiv.org/abs/2503.20084v1), 2025-03-25

(66) [Cross-Tokenizer Distillation via Approximate Likelihood Matching](http://arxiv.org/abs/2503.20083v1), 2025-03-25

(67) [Poor Alignment and Steerability of Large Language Models: Evidence from College Admission Essays](http://arxiv.org/abs/2503.20062v1), 2025-03-25

(68) [BugCraft: End-to-End Crash Bug Reproduction Using LLM Agents in Minecraft](http://arxiv.org/abs/2503.20036v1), 2025-03-25

(69) [OmniNova:A General Multimodal Agent Framework](http://arxiv.org/abs/2503.20028v1), 2025-03-25

(70) [LEGO-Puzzles: How Good Are MLLMs at Multi-Step Spatial Reasoning?](http://arxiv.org/abs/2503.19990v1), 2025-03-25

(71) [ExCoT: Optimizing Reasoning for Text-to-SQL with Execution Feedback](http://arxiv.org/abs/2503.19988v1), 2025-03-25

(72) [CoLLM: A Large Language Model for Composed Image Retrieval](http://arxiv.org/abs/2503.19910v1), 2025-03-25

(73) [Scaling Vision Pre-Training to 4K Resolution](http://arxiv.org/abs/2503.19903v1), 2025-03-25

(74) [A Multi-Agent Framework Integrating Large Language Models and Generative AI for Accelerated Metamaterial Design](http://arxiv.org/abs/2503.19889v1), 2025-03-25

(75) [CausalRAG: Integrating Causal Graphs into Retrieval-Augmented Generation](http://arxiv.org/abs/2503.19878v1), 2025-03-25

(76) [SLA-Awareness for AI-assisted coding](http://arxiv.org/abs/2503.19876v1), 2025-03-25

(77) [Think Twice: Enhancing LLM Reasoning by Scaling Multi-round Test-time Thinking](http://arxiv.org/abs/2503.19855v1), 2025-03-25

(78) [FALCONEye: Finding Answers and Localizing Content in ONE-hour-long videos with multi-modal LLMs](http://arxiv.org/abs/2503.19850v1), 2025-03-25

(79) [Towards Online Multi-Modal Social Interaction Understanding](http://arxiv.org/abs/2503.19851v1), 2025-03-25

(80) [A Comparative Analysis of Word Segmentation, Part-of-Speech Tagging, and Named Entity Recognition for Historical Chinese Sources, 1900-1950](http://arxiv.org/abs/2503.19844v1), 2025-03-25

(81) [FireEdit: Fine-grained Instruction-based Image Editing via Region-aware Vision Language Model](http://arxiv.org/abs/2503.19839v1), 2025-03-25

(82) [ACVUBench: Audio-Centric Video Understanding Benchmark](http://arxiv.org/abs/2503.19951v1), 2025-03-25

(83) [LogQuant: Log-Distributed 2-Bit Quantization of KV Cache with Superior Accuracy Preservation](http://arxiv.org/abs/2503.19950v1), 2025-03-25

(84) [SemEval-2025 Task 9: The Food Hazard Detection Challenge](http://arxiv.org/abs/2503.19800v1), 2025-03-25

(85) [PAVE: Patching and Adapting Video Large Language Models](http://arxiv.org/abs/2503.19794v1), 2025-03-25

(86) [ORION: A Holistic End-to-End Autonomous Driving Framework by Vision-Language Instructed Action Generation](http://arxiv.org/abs/2503.19755v1), 2025-03-25

(87) [Inducing Personality in LLM-Based Honeypot Agents: Measuring the Effect on Human-Like Agenda Generation](http://arxiv.org/abs/2503.19752v1), 2025-03-25

(88) [Optimizing Photonic Structures with Large Language Model Driven Algorithm Discovery](http://arxiv.org/abs/2503.19742v1), 2025-03-25

(89) [Writing as a testbed for open ended agents](http://arxiv.org/abs/2503.19711v1), 2025-03-25

(90) [AdaptiVocab: Enhancing LLM Efficiency in Focused Domains through Lightweight Vocabulary Adaptation](http://arxiv.org/abs/2503.19693v1), 2025-03-25

(91) [BiblioPage: A Dataset of Scanned Title Pages for Bibliographic Metadata Extraction](http://arxiv.org/abs/2503.19658v1), 2025-03-25

(92) [HausaNLP at SemEval-2025 Task 3: Towards a Fine-Grained Model-Aware Hallucination Detection](http://arxiv.org/abs/2503.19650v1), 2025-03-25

(93) [1.4 Million Open-Source Distilled Reasoning Dataset to Empower Large Language Model Training](http://arxiv.org/abs/2503.19633v1), 2025-03-25

(94) [Optimization through In-Context Learning and Iterative LLM Prompting for Nuclear Engineering Design Problems](http://arxiv.org/abs/2503.19620v1), 2025-03-25

(95) [Exploring Next Token Prediction For Optimizing Databases](http://arxiv.org/abs/2503.19619v1), 2025-03-25

(96) [RL-finetuning LLMs from on- and off-policy data with a single algorithm](http://arxiv.org/abs/2503.19612v1), 2025-03-25

(97) [Innate Reasoning is Not Enough: In-Context Learning Enhances Reasoning Large Language Models with Less Overthinking](http://arxiv.org/abs/2503.19602v1), 2025-03-25

(98) [HoarePrompt: Structural Reasoning About Program Correctness in Natural Language](http://arxiv.org/abs/2503.19599v1), 2025-03-25

(99) [The Greatest Good Benchmark: Measuring LLMs' Alignment with Utilitarian Moral Dilemmas](http://arxiv.org/abs/2503.19598v1), 2025-03-25

(100) [Context-Efficient Retrieval with Factual Decomposition](http://arxiv.org/abs/2503.19574v1), 2025-03-25
